src/app/compare/Compare.tsx

```diff
@@ -119,6 +120,11 @@ export default function Compare({ session }: { session: CompareSession }) {
   const hasItems = compareItems.length > 0;
   useEffect(() => {
     if (!hasItems) {
+      showNotification({
+        type: 'warning',
+        title: t('Compare.Error.Invalid'),
+        body: session.query,
```

Comment:
Not sure if I should include this but session doesn't have any other info about the item attempting to be compared.